RNA-Seq Analysis/README.md

```diff
@@ -12,12 +12,17 @@ Find an RNA-Seq dataset and perform differential gene expression analysis to det
 If you do not see the information about the biological conditions of the samples in your experiment, you can check SRA for that information. Here is the SRA page for ERP010786: https://www.ncbi.nlm.nih.gov/Traces/study/?acc=ERP010786
 
 Then, do the following in R Markdown:
-1. Use DESeq2 for the differential gene expression analysis. Find the DEGs between your conditions of interest. 
+1. Use DESeq2 for the differential gene expression analysis. Find the DEGs between your conditions of interest.
+	- Make sure to add an interactable table of DEGs using a package such as [DT](https://rstudio.github.io/DT/)
 2. Create a PCA plot colored by the condition of interest
 3. Create a Volcano Plot
-4. Create a heatmap showing the top 20 over- and under-expressed DEGs
+4. Create a heatmap with the top 10 over-expressed DEGs, and top 10 under-expressed DEGs
+	- Top is defined by DEGs having the lowest adjusted p-values, where the absolute log2 fold change is > 1
 5. Do GSEA on the results and plot the top 5 pathways
+	- Make sure you use an appropriate [ranking metric](https://www.gsea-msigdb.org/gsea/doc/GSEAUserGuideTEXT.htm#_Metrics_for_Ranking) (other metrics outside this document are also acceptable if justified)
```

Comment:
I think we should probably explicitely tell them to use the wald `stat` column for this